src/index.html

```diff
@@ -6,9 +6,85 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+      href="#home"
+      class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#aplle"
+            class="nav__link is-active">
+            Apple
+            </a>
```

Comment:
1) no needs to write attributes with a new line if you have 2 or less attributes
2) apply code styling recommendations from [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
```